src/python/dxpy/utils/exec_utils.py

```diff
@@ -23,6 +23,7 @@
 import os, sys, json, re, collections, logging, argparse, string, itertools, subprocess, tempfile
 from functools import wraps
 from collections import namedtuple
+import pipes
```

Comment:
I think the change in this file is appropriate, but I don't know about the other two files. Do you want to wait until you have a fix for dx-unpack, or do you want to just land this part now?